vuepress/docs/next/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -522,31 +453,33 @@ above would look like this:
           // note that property's fields are not mentioned - defaulting them to 12 columns in are that is available to them
```

Comment:
'12 columns in are that'-- unsure what is meant here

Reply:
LOL good catch - i didn't even read this somehow?? absolutely no clue what this comment addresses or how to interpret it. @nshaw

Reply:
@Lyd1aCla1r3 your edit looks good. A lot of modern UX scaffolding divides the page into 12 columns.